packages/opentelemetry-sdk-node/src/types.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ContextManager } from '@opentelemetry/context-base';
+import type { api, core, metrics, node, resources, tracing } from '.';
```

Comment:
nice! `import type` is finally available

Reply:
oh something new, nice :)